queue_job/jobrunner/runner.py

```diff
@@ -22,13 +22,33 @@
 How to use it?
 --------------
 
-* Set the following environment variables:
+* Optionally adjust your configuration through environment variables:
 
-  - ``ODOO_CONNECTOR_CHANNELS=root:4`` (or any other channels configuration)
-  - optional if ``xmlrpc_port`` is not set: ``ODOO_CONNECTOR_PORT=8069``
+  - set ``ODOO_CONNECTOR_CHANNELS=root:4`` (or any other channels
+    configuration) if you don't want the default ``root:1``.
+
+  - if ``xmlrpc-port`` is not set, you can set it for the jobrunner only with:
+    ``ODOO_CONNECTOR_PORT=8069``.
+
+* Alternatively, configure the channels through the Odoo configuration
+  file, like:
+
+.. code-block:: ini
+
+  [options-queue_job]
```

Comment:
I wonder if mixing `-` and `_` wouldn't bring some confusion for users. I understand that `queue_job` represents the name of the addon, but maybe `options-queue-job` would be simpler? What do you think?